queries.sql

```diff
@@ -22,3 +22,93 @@ WHERE name <> 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- First transaction
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
```

Comment:
- [ ] According to the project requirement, we should verify that the change was made and rollbacked. Therefore I suggest also checking  if the change was reversed by adding `SELECT * FROM animals;` after the **ROLLBACK** keyword

Reply:
Done, thank you for the review 👍😃